modules/administration-guide/pages/installing-che-on-codeready-containers.adoc

```diff
@@ -10,10 +10,69 @@
 
 :context: installing-{prod-id-short}-on-codeready-containers
 
-This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers. To install {prod-short}, use the`{prod-cli}` command-line tool or the OpenShift web console GUI.
+This chapter describes the creation of a single-node OpenShift cluster for deploying a {prod-short} instance using CodeReady Containers.
 
-include::partial$proc_using-chectl-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+.Prerequisites
 
-include::partial$proc_using-openshift-web-console-to-install-che-on-codeready-containers.adoc[leveloffset=+1]
+* The `{prod-cli}` management tool is available. See xref:installing-the-chectl-management-tool.adoc[].
+* An installation of CodeReady Containers. See link:https://console.redhat.com/openshift/create/local[Installing CodeReady Containers].
+* The path to the user's CodeReady Containers pull secret is known as __<pull-secret-file>__.
+* Available memory: 12 GB or more.
+* Available disk size: 64 GB or more.
 
-:context: {parent-context-of-installing-che-on-codeready-containers}
+.Procedure
+
+. Remove any previous cluster:
++
+----
```

Comment:
```suggestion
[source,terminal] 
----
```
Source blocks should have the language specified when appropriate (e.g. [source,yaml]). Terminal commands/output should have [source,terminal] specified.

Reply:
Many instances.